### PR TITLE
chore: B14S6 beta to master merge

### PR DESCRIPTION
**Pull request by mcdurdin**
Doing this early to test the patch for GitHub comments

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -13,6 +13,113 @@
...
+* chore(windows/resources): Address more TODO links for help (#4351)
+* fix(windows): setup strings comment for language (#4362)
+* fix(windows): Splash button sizes (#4348)
+* chore: manual version increment (#4363)
+
+## 14.0.228 beta 2021-01-22
+
+* fix(web): uses CSS line-height to vertically center oversized OSK keycaps (#4255)
+* fix(web): OSK loading efficiency (#4279)
+
```
Comment (darcywong00) [inline HISTORY.md:122]: Shouldn't the 14.0 beta changes come after l. 244? (where 15.0 begins)

File: HISTORY.md [hunk 2]
```diff
@@ -218,12 +242,6 @@
 
 * fix(web): uses CSS line-height to vertically center oversized OSK keycaps (#4255)
 * fix(web): OSK loading efficiency (#4279)
-* fix(web): default popup key selection, space highlight after popup (#4306)
-* fix(web): language menu key highlighting (#4308)
-* fix(web): dynamic font downscaling for OSK keys (#4270)
-* chore(windows/resources): Cleanup Notes and Tips in help (#4307)
-* fix(developer): Debug character grid performance (#4237)
-* chore(android/samples): Remove old sample keyboard loaded code (#4315)
```
Comment (darcywong00) [inline HISTORY.md:226]: Did we lose some 14.0.228 history?

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.16-alpha](https://keyman.com/downloads/releases/alpha/15.0.16)

**Pull request merged**